common/src/stack/command/stack/commands/add/pallet/__init__.py

```diff
@@ -326,13 +326,21 @@ def run(self, params, args):
 		paths_to_args = {data['exploded_path']: data['canonical_arg'] for data in pallet_args.values()}
 
 		# we have everything we need, copy the pallet to the fs, add it to the db, and maybe patch it
+		self.beginOutput()
 		for pallet in flatten(pallet_infos.values()):
 			self.copy(stacki_pallet_dir, pallet, clean)
 			self.write_pallet_xml(stacki_pallet_dir, pallet)
 			if updatedb:
 				self.update_db(pallet, paths_to_args[pallet.pallet_root])
 			if stacki_pallet_dir == '/export/stack/pallets':
 				self.patch_pallet(pallet)
+			output = info_getter(pallet)
+			self.addOutput(output[0], (output[1],output[2],output[4],output[3]))	
```

Comment:
Since we know what fields you want from the pallet object (based on the header), maybe just put those here explicitly:

```
self.addOutput(pallet.name, (pallet.version, pallet.release, pallet.arch, pallet.distro_family))
```

That way the code is a little more clear, and you don't need the call to `info_getter()`

Reply:
thank you ... still finding my way through the code/data. updated